Project.toml

```diff
@@ -55,13 +55,13 @@ DiffEqDevTools = "2.44"
 FastAlmostBandedMatrices = "0.1.1"
 FastClosures = "0.3"
 ForwardDiff = "0.10.36"
-JET = "0.8"
```

Comment:
any idea why compathelper didn't pick it up?

Reply:
I think CompatHelper only helps with compats for dependencies right? Since we are using JET in testing, CompatHelper would not be able to pick them.

Reply:
We might have to do someething with subdirs? @ArnoStrouwen has done this thing before 
https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/.github/workflows/CompatHelper.yml#L26

Reply:
https://github.com/JuliaRegistries/CompatHelper.jl/issues/166